### ISSUE TITLE
lookups.v1.phone_numbers(..).fetch(..) fails with ArgumentError

### ISSUE THREAD

**Issue by engshien** [twilio/twilio-ruby#518]
### Issue Summary
lookups.v1.phone_numbers(..).fetch(..) fails with ArgumentError in [generated code](https://github.com/twilio/twilio-ruby/blob/704ed3d6cfd2cd7038511b75089a66a57e2051a3/lib/twilio-ruby/rest/lookups/v1/phone_number.rb#L105
)

### Steps to Reproduce
Run code snippet

### Code Snippet
```ruby
require 'twilio-ruby'
account_sid = ...
auth_token = ...
twilio_client = Twilio::REST::Client.new(account_sid, auth_token)
result = twilio_client.lookups.v1.phone_numbers('+551155256325').fetch(type: 'carrier')
```

### Exception/Log
```
wrong number of arguments (given 3, expected 2)
/Users/ewu/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/twilio-ruby-5.39.0/lib/twilio-ruby/framework/version.rb:60:in `fetch'
/Users/ewu/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/twilio-ruby-5.39.0/lib/twilio-ruby/rest/lookups/v1/phone_number.rb:105:in `fetch'
```

### Technical details:
* twilio-ruby version: twilio-ruby-5.39.0
* ruby version: 2.6.6

**Comment by engshien** [comment]
I think `fetch(.., params = {} ...)`  is now `fetch(..., params: {} ...)`.

**Comment by eshanholtz** [comment]
Nice catch. Unfortunately this was a bug introduced during a refactor done to support custom header parameters in the underlying API in #515 and #516. I tried to ensure backwards compatibility, but apparently none of our autogenerated integration tests for fetch have `params` passed in. I will work on a fix for this and release a patched version. Thanks!

**Comment by eshanholtz** [comment]
Fix released in patch [version 5.39.1](https://github.com/twilio/twilio-ruby/releases/tag/5.39.1)
